internal/jimm/bootstrap/bootstrap.go

```diff
@@ -577,6 +577,7 @@ func (b *bootstrapManager) consumeCommandOutput(ctx context.Context, outputCh <-
 			b.writeJobLog(ctx, jobId, output.Err.Error())
 			return errors.E(fmt.Errorf("command failed: %w", output.Err))
 		}
+		zapctx.Debug(ctx, "command output", zap.String("line", output.Line))
```

Comment:
I'd like to keep this as when in debug it's helpful to also see the logs on JIMM's side and not go to DB to see if it is running.

Reply:
What I mean is, you can keep the debug logging but it should be done here. It should be done higher up. Otherwise you fall victim to logging at different layers and logging the same thing multiple times. So at the point where we no longer return the error, I.e. The place where we are saving it somewhere, we can also log it.

Reply:
Ahh you mean at the call site. Not sure how I'd do it higher up because we're reading from the chan here.

Reply:
Fair point and we'll eventually test this in parallel, I've added the job id to the log.